scripts/create-database.js

```diff
@@ -0,0 +1,57 @@
+// require the pg package
```

Comment:
The comments are quite verbose here, you can remove them and condense it all down a bit

Also, its always a good idea to run prettier over your files before submission/set it up so it runs automatically on saving a file. You are missing the semicolons on the end of most lines here